rest_framework/filters.py

```diff
@@ -45,6 +45,7 @@ class SearchFilter(BaseFilterBackend):
         '=': 'iexact',
         '@': 'search',
         '$': 'iregex',
+        '&': 'unaccent',
```

Comment:
can you please add relevant test for this search feature? or they are already covered?

Reply:
@auvipy hey! echoing a bit what [I mentioned last year](https://github.com/encode/django-rest-framework/pull/7733#issuecomment-919926794):
This feature is only for Django's PostgreSQL backend, which does not include tests in this repo.
Adding tests could be more complex than the scope of this PR, I think it's a perfectly valid issue to open, but would not block this.

Reply:
https://github.com/encode/django-rest-framework/blob/master/tests/test_filters.py#L52 not possible to add any tests here right?